README.md

```diff
@@ -178,7 +178,7 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 }  
 ```
 
-For a guide on Reconcilers, Clients, and interacting with resource Events, see the [Client API doc][doc_client_api].
+For a guide on Reconcilers, Clients, and interacting with resource Events, see the [Kubebuilder Book][doc_kubebuilder_book].
```

Comment:
Revert this one and point out to https://sdk.operatorframework.io/docs/golang/references/client/